src/models/transactions/escrowFinish.ts

```diff
@@ -14,10 +14,9 @@ export interface EscrowFinish extends BaseTransaction {
  * Verify the form and type of an EscrowFinish at runtime.
  *
  * @param tx - An EscrowFinish Transaction.
- * @returns Void.
  * @throws When the EscrowFinish is Malformed.
  */
-export function verifyEscrowFinish(tx: EscrowFinish): void {
+export function verifyEscrowFinish(tx: Record<string, unknown>): void {
```

Comment:
Why are we getting rid of EscrowFinish here? (Curious)

Reply:
Because this method is only going to be used by JS folks. So we don't have any of the benefit of typing anyways, and we shouldn't be typing it as such.